client/app/telecom/overTheBox/tasks/overTheBox-tasks.controller.js

```diff
@@ -28,6 +28,11 @@ angular.module("managerApp").controller("OverTheBoxTasksCtrl", function ($transl
         return OvhApiOverTheBox.Lexi().getTasks({ serviceName: self.serviceName }).$promise.then(
             function (taskIds) {
                 self.taskIds = taskIds;
+                self.tasks = [];
+
+                for (const id of self.taskIds) {
```

Comment:
Maybe this is missing the point of lazy loading inside the datagrid, inPlace variable manipulation could be avoided in those cases. ```task = self.taskIds.map((id)=>this.getTask(id))``` would be a lot nicer.